v2/index.html

```diff
@@ -2122,10 +2122,47 @@ <h5>Repeated measures</h5>
             </aside>
         </section>
         <section>
-            <h5>Rhythmic sequence</h5>
+            <h5>Duration Group</h5>
             <p>
-                When the same rhythmic sequence is repeated, the sequence of rhythmic values can be stated once
-                before the note names.
+                Multiple duration values MAY be declared together. These values form a Duration Group, and MAY be
+                used to indicate repeating rhythmic patterns for the notes that follow.
+            </p>
+            <p>
+                The number and order of duration values MUST apply to an equivalent number of notes that follow.
+                If the number of notes that follow exceeds the number of duration values given in the Duration
+                Group, the duration values MUST repeat from the first.
+            </p>
+            <p>
+                A duration group MUST remain in effect until another duration value is declared. A new duration value
```

Comment:
```suggestion
                A duration group remains in effect until another duration value is declared. A new duration value
```

Reply:
Why did you remove this requirement? Without it, there is no requirement that a duration value interrupts the repetition of the group.

Reply:
Same as above. I find this confusing.

Reply:
Could you suggest some other wording that keeps the normative requirement that a duration group remains in effect until cancelled?

Reply:
Something like `"A duration group indicates that the note names following it have a duration as given in the group and in that order. When the number of notes reaches the number of durations given in the group, the durations applies again from the beginning of the group. Any duration given in the notes following the duration group cancels it."

Reply:
I think we still need the MUST in there to indicate the requirement.